packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx

```diff
@@ -159,6 +163,13 @@ export class AtomicSearchBox {
    */
   @Prop({reflect: true}) public clearFilters = true;
 
+  /**
+   * Whether to interpret advanced [Coveo Cloud query syntax](https://docs.coveo.com/en/1814/searching-with-coveo/search-prefixes-and-operators) in the query.
+   *
+   * When the `redirection-url` property is set and redirects to a page with more `atomic-search-box` components, all `atomic-search-box` components need to have the same value for `enable-query-syntax`.
```

Comment:
Should we notify about the lack of syntax highlighting, users of the JSUI will wonder if it's available?

Reply:
Yeah I see why you didn't include that in the description; no need to get into such details.

Reply:
I would add a bit more meat maybe ?

Something that explains the drawback of enabling query syntax: https://github.com/coveo/ui-kit/pull/2642#discussion_r1064720365

It's an option that most implementers should not enable, otherwise will have a negative effect on their UX/relevancy.

"You should know what you are doing if you plan to use this" type of thing.

Reply:
Good idea Oli, I'll tweak my suggestion above.

Reply:
If we see people surprised by the lack of syntax highlighting, then maybe we should add that to the description. Until then, I prefer not to document the lack of a feature.